service/add_email_test.go

```diff
@@ -13,6 +13,29 @@ import (
 	"testing"
 )
 
+func TestService_GetEmail_Error(t *testing.T) {
+	const (
+		testEmail        = "test@email.com"
+		userId    uint64 = 2342342
+	)
+
+	mockDB, mock, _ := sqlmock.New()
+	mock.ExpectExec(`SELECT FROM emails*`).WithArgs(sqlmock.AnyArg(), testEmail, userId).
+		WillReturnResult(driver.ResultNoRows)
+
+	s := NewService(mockDB)
+
+	res, err := s.AddEmail(context.Background(), &email.AddEmailRequest{
```

Comment:
Change this to test the RemoveEmail method instead of the AddEmail method. You'll need to adapt the rest of the test function around this.